patient-dao/src/main/java/hu/kuncystem/patient/dao/user/JDBCUserGroupDao.java

```diff
@@ -69,6 +70,8 @@ public UserGroup mapRow(ResultSet rs, int row) throws SQLException {
     public List<User> getAllUserFromGroup(UserGroup group) {
         try {
             return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        } catch (EmptyResultDataAccessException e) {
+            return null;
         } catch (DataAccessException e) {
             e.printStackTrace();

```

Comment:
Do not you want to throw an exception here? I would not catch this one, instead we should throw it away and let the service handle this. Probably we want to notify the customers on the GUI that we are facing a DAE (with some indicator; i.e. please try later). This is something we should consider in the DAO layer generally.